.sauce/config.yml

```
@@ -21,7 +21,7 @@ suites:
   - name: "iPhone-Pro"
     devices:
       - name: "iPhone 13 Pro.*"
-        platformVersion: "15.6"
+        platformVersion: "15.7"
```

What about using. So it will just pick any 15.x device and the tests won't break that often?
```suggestion
        platformVersion: "15"
```

---

It doesn't just use 15.0 then? If so, this seems like a good idea!

---

Didn't know that this works. Perfect!